appengine/flexible/websockets/requirements.txt

```diff
@@ -2,4 +2,4 @@ Flask==1.1.4 # it seems like Flask-sockets doesn't play well with 2.0+
 Flask-Sockets==0.2.1
 gunicorn==20.1.0
 requests==2.27.1
-markupsafe==2.0.1
+markupsafe===2.0.1
```

Comment:
Hi @engelke  is the extra = intentional? Maybe a convention I haven't seen before.